src/reducer.spec.ts

```diff
@@ -39,35 +44,86 @@ test('reducer should store reducer function', t => {
 test(
   'combineReducers should reduce actions to state',
   marbles(m => {
-    const source = m.hot('  121', inputs);
-    const expected = m.hot('245', outputs);
+    const action$ = m.hot('  121', actions);
+    const expected$ = m.hot('245', numbers);
+
+    m.expect(
+      action$.pipe(combineReducers(1, [handleOne, handleMany]))
+    ).toBeObservable(expected$);
+  })
+);
+
+test(
+  'combineReducers should reduce state from other streams',
+  marbles(m => {
+    const action$ = m.hot('  --');
+    const word$ = m.hot('    ab', words);
+    const expected$ = m.hot('24', numbers);
+
+    const handleWord = reducer(
+      word$,
+      (state: number, word) => state + word.length
+    );
+
+    m.expect(action$.pipe(combineReducers(1, [handleWord]))).toBeObservable(
+      expected$
+    );
+  })
+);
+
+test(
+  'combineReducers should support both actions and other streams',
+  marbles(m => {
+    const action$ = m.hot('  -1-', actions);
+    const word$ = m.hot('    a-b', words);
+    const expected$ = m.hot('235', numbers);
+
+    const handleWord = reducer(
+      word$,
+      (state: number, word) => state + word.length
+    );
 
     m.expect(
-      source.pipe(combineReducers(1, [handleOne, handleMany]))
-    ).toBeObservable(expected);
+      action$.pipe(combineReducers(1, [handleOne, handleWord]))
+    ).toBeObservable(expected$);
   })
 );
 
 test(
   'combineReducers should not silence errors',
   marbles(m => {
-    const source = m.hot('  1d1', inputs);
-    const expected = m.hot('2# ', outputs);
+    const action$ = m.hot('  1d1', actions);
+    const expected$ = m.hot('2# ', numbers);
 
     m.expect(
-      source.pipe(combineReducers(1, [handleOne, handleDecrement]))
-    ).toBeObservable(expected);
+      action$.pipe(combineReducers(1, [handleOne, handleDecrement]))
+    ).toBeObservable(expected$);
   })
 );
 
 test(
   'combineReducers should handle reducers for multiple actions',
   marbles(m => {
-    const source = m.hot('  1', inputs);
-    const expected = m.hot('5', outputs);
+    const action$ = m.hot('  12', actions);
+    const expected$ = m.hot('55', numbers);
 
-    m.expect(source.pipe(combineReducers(1, [alwaysReset]))).toBeObservable(
-      expected
+    m.expect(action$.pipe(combineReducers(1, [alwaysReset]))).toBeObservable(
+      expected$
     );
   })
 );
+
+test(
+  'combineReducers should not emit before after the seed stream does',
```

Comment:
```suggestion
  'combineReducers should not emit and ignore actions until the seed stream emits a value',
```

^ would this make it a bit more clear?